src/services/swap_service.ts

```diff
@@ -136,7 +136,7 @@ export class SwapService {
         const breakdown: GetSwapQuoteResponseLiquiditySource[] = [];
         return Object.entries(sourceBreakdown).reduce((acc: GetSwapQuoteResponseLiquiditySource[], [source, percentage]) => {
             return [...acc, {
-                name: source,
+                name: source === 'Native' ? '0x' : source,
```

Comment:
@dekz and I were talking about updating this in swapper too. Is there an enum we can use that is either exported by asset-swapper or otherwise that will cause this compilation to fail if we change the string?